### PR TITLE
refresh lightbox

### PR DESCRIPTION
**Pull request by spasovski**
Couldn't find an associated bug number. Freshly rebased and made clicking on background dismiss the lightbox again.

### CODE REVIEW

File: src/media/css/lightbox.styl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'lib';
 
 #lightbox {
-    background: rgba(0,0,0,.75);
+    background: rgba(198,198,198,.8);
```
Comment (ngokevin) [inline src/media/css/lightbox.styl]: use a color palette color like https://github.com/mozilla/marketplace-elements/blob/master/marketplace-elements.styl#L294

````rgba($greyscale-grey, 0.8)```

File: src/media/img/pretty/close_lightbox.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ngokevin) [inline src/media/img/pretty/close_lightbox.svg]: is this the red one or blue one?

**Comment by ngokevin** [comment]
Screenshot not centered, and the close button icon isn't in the top right.

screen shot 2015-01-20 at 6 42 03 pm

**Comment by ngokevin** [comment]
If it's trivial, it'd be nice to disable/grey the buttons instead of hiding them in case I get to the end, the button disappears, I accidentally click on the bg, and the lightbox disappears.

**Comment by ngokevin** [comment]
don't know if @pwalm has seen this, but ccing him

**Comment by ngokevin** [comment]
r+wc

**Comment by pwalm** [comment]
@ngokevin Totally right, we should disable instead of hiding. Can we increase the opacity on that background a bit? Looks a bit busy when you see the background like that.

**Comment by mstriemer** [comment]
Looks like this needs a rebase with master after the merge.

**Comment by spasovski** [comment]
Going to reopen this later with comments addressed and a rebase. Thanks for the input folks.

**Pull request closed without merging**